README.md

```diff
@@ -52,7 +52,7 @@ Persistent settings
   ☑ Enable Telemetry
 ```
 2. Save the Advanced options.
-3. Operating System `CHOOSE OS -> Use custom` and pick the `2021-03-04-raspios-buster-arm64-lite.zip` you downloaded.
+3. Operating System `CHOOSE OS -> Use custom` and pick the `2021-05-07-raspios-buster-armhf-lite.zip` you downloaded.
```

Comment:
can you update based on my pull request at https://github.com/curl-bash/curlbash/pull/11?

Reply:
I would if I could figure out how to do it with GitHub's convoluted UI. 🤷‍♂️